tests/api.c

```diff
@@ -65808,6 +65808,1036 @@ static int test_override_alt_cert_chain(void)
+    const char* certfile_s, int fmt_cs, /* server cert file path and format */
+    const char* pkey_c,     int fmt_kc, /* client private key and format */
+    const char* pkey_s,     int fmt_ks  /* server private key and format */
+    )
+{
+    int ret;
+    if (ctx_c != NULL && *ctx_c == NULL) {
+        *ctx_c = wolfSSL_CTX_new(method_c());
+        if (*ctx_c == NULL) {
+            return -1;
+        }
+        wolfSSL_CTX_set_verify(*ctx_c, WOLFSSL_VERIFY_PEER, NULL);
+
+        ret = wolfSSL_CTX_load_verify_locations(*ctx_c, caCertFile, 0);
+        if (ret != WOLFSSL_SUCCESS) {
+            return -1;
+        }
+        wolfSSL_SetIORecv(*ctx_c, test_memio_read_cb);
+        wolfSSL_SetIOSend(*ctx_c, test_memio_write_cb);
+
+        ret = wolfSSL_CTX_use_certificate_file(*ctx_c, certfile_c, fmt_cc);
+        if (ret != WOLFSSL_SUCCESS) {
+            return -1;
+        }
+        ret = wolfSSL_CTX_use_PrivateKey_file(*ctx_c, pkey_c, fmt_kc);
+        if (ret != WOLFSSL_SUCCESS) {
+            return -1;
+        }
+    }
+
+    if (ctx_s != NULL && *ctx_s == NULL) {
+        *ctx_s = wolfSSL_CTX_new(method_s());
+        if (*ctx_s == NULL) {
+            return -1;
+        }
+        wolfSSL_CTX_set_verify(*ctx_s, WOLFSSL_VERIFY_PEER, NULL);
+
+        ret = wolfSSL_CTX_load_verify_locations(*ctx_s, cliCertFile, 0);
+        if (ret != WOLFSSL_SUCCESS) {
+            return -1;
+        }
+
+        ret = wolfSSL_CTX_use_PrivateKey_file(*ctx_s, pkey_s, fmt_ks);
+        if (ret != WOLFSSL_SUCCESS) {
+            return -1;
+        }
+        ret = wolfSSL_CTX_use_certificate_file(*ctx_s, certfile_s, fmt_cs);
+        if (ret != WOLFSSL_SUCCESS) {
+            return -1;
+        }
+        wolfSSL_SetIORecv(*ctx_s, test_memio_read_cb);
+        wolfSSL_SetIOSend(*ctx_s, test_memio_write_cb);
+        if (ctx->s_ciphers != NULL) {
+            ret = wolfSSL_CTX_set_cipher_list(*ctx_s, ctx->s_ciphers);
+            if (ret != WOLFSSL_SUCCESS) {
+                return -1;
+            }
+        }
+    }
+
+    if (ctx_c != NULL && ssl_c != NULL) {
+        *ssl_c = wolfSSL_new(*ctx_c);
+        if (*ssl_c == NULL) {
+            return -1;
+        }
+        wolfSSL_SetIOWriteCtx(*ssl_c, ctx);
+        wolfSSL_SetIOReadCtx(*ssl_c, ctx);
+    }
+    if (ctx_s != NULL && ssl_s != NULL) {
+        *ssl_s = wolfSSL_new(*ctx_s);
+        if (*ssl_s == NULL) {
+            return -1;
+        }
+        wolfSSL_SetIOWriteCtx(*ssl_s, ctx);
+        wolfSSL_SetIOReadCtx(*ssl_s, ctx);
+#if !defined(NO_DH)
+        SetDH(*ssl_s);
+#endif
+    }
+
+    return 0;
+}
+#endif /* HAVE_RPK */
+
+static int test_rpk_set_xxx_cert_type(void)
+{
+    int ret = TEST_SKIPPED;
+#if defined(HAVE_RPK)
+
+    char ctype[MAX_CLIENT_CERT_TYPE_CNT + 1];   /* prepare bigger buffer */
+    WOLFSSL_CTX* ctx = NULL;
+    WOLFSSL* ssl = NULL;
+    int tp;
+
+    ctx = wolfSSL_CTX_new(wolfTLSv1_3_client_method());
+    AssertNotNull(ctx);
```

Comment:
Use 'Expect' APIs instead of 'Assert' versions If possible.

Reply:
Replaced 'Assert' APIs with 'Expect' ones.